okd/install-packages.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+#Script to install all packages to run the Keylime agent
+
+rpm-ostree cleanup -m
+rpm-ostree upgrade
+
+# Updating base packages
+
+curl -L -O http://download-node-02.eng.bos.redhat.com/fedora/linux/updates/32/Everything/x86_64/Packages/d/dbus-libs-1.12.20-1.fc32.x86_64.rpm
+
+curl -L -O http://download-node-02.eng.bos.redhat.com/fedora/linux/updates/32/Everything/x86_64/Packages/g/glib2-2.64.3-2.fc32.x86_64.rpm 
+
+curl -L -O http://download-node-02.eng.bos.redhat.com/fedora/linux/updates/32/Everything/x86_64/Packages/p/pcre2-10.35-3.fc32.x86_64.rpm
+
+curl -L -O http://download-node-02.eng.bos.redhat.com/fedora/linux/updates/32/Everything/x86_64/Packages/p/pcre2-syntax-10.35-3.fc32.noarch.rpm 
+
+rpm-ostree override replace ./dbus-libs-1.12.20-1.fc32.x86_64.rpm ./glib2-2.64.3-2.fc32.x86_64.rpm ./pcre2-syntax-10.35-3.fc32.noarch.rpm ./pcre2-10.35-3.fc32.x86_64.rpm
+
+# Installing packages
+
+rpm-ostree install automake \
+dbus-devel \
+dnf-plugins-core \
```

Comment:
This seems like an odd one. We aren't using `dnf` at all but rpm-ostree. Was this just needed in the keylime-agent docker file? Or is there something else in the package that's needed for this?

Reply:
I'm not too sure about this. I was following the list of dependencies from the Dockerfile - 
https://github.com/redhat-et/edge-security-demo/blob/14fc90218c1a6fe5ec3bb60127e960f9c8dfabf4/docker/Dockerfile#L17 @lukehinds can help us on this.

Reply:
we definitely don't need automake

we do need dbus-devel . i don't think we need dnf-plugins-core

Reply:
> we definitely don't need automake

We need it to build tpm_server right? After it's built we can distribute the binary to the nodes but we need at least one node to build it on, so I'd leave it in.